setup.cfg

```diff
@@ -1,6 +1,6 @@
 [metadata]
 name = cmsplugin-section-ai
-version = 0.0.2
+version = 1.0.0
```

Comment:
Bumping up the minor version should also be ok since the version is pre 1.0 and therefore minor version bumps can also include breaking changes. But if you feel things will start staying more stable then 1.0 might be reasonable.

Reply:
Good point. I might go with `0.1.0` at this point. It's starting to get more stable and has more features. I'll change that